client/containers/DashboardSubmissions/SubmissionItems.tsx

```diff
@@ -68,39 +85,31 @@ const SubmissionItems = (props: Props) => {
 		},
 	});
 
+	const [localStats, setLocalStats]: [{ [pubId: string]: SubmissionStatus }, any] = useState({});
 	const canLoadMorePubs = !hasLoadedAllPubs;
 	useInfiniteScroll({
 		enabled: !isLoading && canLoadMorePubs,
 		useDocumentElement: true,
 		onRequestMoreItems: loadMorePubs,
 	});
 
-	const renderPubs = () => {
-		return pubs.map((pub) => (
-			<PubOverviewRow
-				pub={pub}
-				key={pub.id}
-				leftIconElement="manually-entered-data"
-				hasSubmission={true}
-			/>
-		));
+	const trackLocalStatus = (pubId: string, status?: string) => {
```

Comment:
Perhaps `updateLocalStatus` would be a better name — I don't know whether "track" is a read-ish or write-ish operation from a glance. Also, this can be wrapped in a `useCallback()` and `status` could be narrowed to `SubmissionStatus`.